README.md

```diff
@@ -2,54 +2,61 @@
 
 # Azure API Management Extension for Visual Studio Code (Preview)
 
-[Azure API Management](https://aka.ms/apimrocks) is a fully managed service that enables customers to publish, secure, transform, maintain, and monitor APIs. Use the Azure API Management extension for VS Code to quickly manage your service instances and APIs.
+Use the Azure API Management extension to perform common management operation on your Azure API Management service instances without switching away from Visual Studio Code.

```

Comment:
should it be "common management **operations**"

Reply:
My bad. Fixed and resubmitted.